App/UI/Privacy/PrivacyVC.swift

```diff
@@ -79,7 +79,7 @@ final class PrivacyVC: ViewControllerWithLocalState<PrivacyView> {
 
     guard
       let cell = checkBoxCell,
-      cell.convert(cell.bounds.origin, to: nil).y < self.rootView.frame.height
+      cell.convert(cell.bounds.origin, to: nil).y + cell.bounds.size.height < self.rootView.frame.height - self.rootView.scrollableGradientView.bounds.size.height
```

Comment:
Please split this into two lines to avoid the swift lint warning:
```
Line Length Violation: Line should be 130 characters or less: currently 162 characters (line_length)
```

Reply:
Done. Sorry about that.